spec/arns_spec.lua

```diff
@@ -424,6 +424,56 @@ describe("arns", function()
 		end)
 	end
 
+	describe("getTokenCost", function()
+		it("should return the correct token cost for a lease", function()
+			local baseFee = 500000000
+			local years = 2
+			local demandFactor = 1.0
```

Comment:
Test might have slightly higher value if this were not 1.

Reply:
sure

Reply:
[c1e270f](https://github.com/ar-io/ar-io-network-process/pull/108/commits/c1e270f808b4d5da536a70d46de42ddafb0a0e00)